jupyter/src/askee_kernel/askeekernel.py

```diff
@@ -218,6 +218,9 @@ def format_resp_value(v):
                                     float(val['size']),
                                     val['bins'])
 
+        if ty == 'latex':
+            return { 'text/latex': val }
```

Comment:
for example, maybe this is where we can bling "val" if needed

Reply:
I've migrated the blinging to `askeekernel.py`.